### PR TITLE
Update branch references in the README

### PR DESCRIPTION
**Pull request by dnrce**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,5 +143,5 @@ NervesHub server via a Phoenix Channel and can be used to exercise the server
 for development and test.
 
 See the
-[nerves_hub/README.md](https://github.com/nerves-hub/nerves_hub/blob/next/README.md)
+[nerves_hub/README.md](https://github.com/nerves-hub/nerves_hub/blob/master/README.md)
```
Comment (fhunleth) [inline README.md:146]: Oh wow, this section is dated. I think we want to point people to https://github.com/nerves-hub/nerves_hub_link, but that's not quite right either. @jjcarstens - want to leave this for now and figure it out in a future PR or is there an easy fix?

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # nerves_hub_www
 
-[![CircleCI](https://circleci.com/gh/nerves-hub/nerves_hub_web/tree/next.svg?style=svg)](https://circleci.com/gh/nerves-hub/nerves_hub_web/tree/next)
-[![Coverage Status](https://coveralls.io/repos/github/nerves-hub/nerves_hub_web/badge.svg?branch=next)](https://coveralls.io/github/nerves-hub/nerves_hub_web?branch=next)
+[![CircleCI](https://circleci.com/gh/nerves-hub/nerves_hub_web/tree/next.svg?style=svg)](https://circleci.com/gh/nerves-hub/nerves_hub_web/tree/main)
+[![Coverage Status](https://coveralls.io/repos/github/nerves-hub/nerves_hub_web/badge.svg?branch=next)](https://coveralls.io/github/nerves-hub/nerves_hub_web?branch=main)
```
Comment (fhunleth) [inline README.md:4]:
```suggestion
[![CircleCI](https://circleci.com/gh/nerves-hub/nerves_hub_web/tree/main.svg?style=svg)](https://circleci.com/gh/nerves-hub/nerves_hub_web/tree/main)
[![Coverage Status](https://coveralls.io/repos/github/nerves-hub/nerves_hub_web/badge.svg?branch=main)](https://coveralls.io/github/nerves-hub/nerves_hub_web?branch=main)
```

There were two more references to next that I updated above.

File: README.md [hunk 3]
```diff
@@ -11,9 +11,9 @@ server.
 The public NervesHub instance at `nerves-hub.org` will be turned off on March
 31st, 2022.  See [NervesHub Sunset](https://elixirforum.com/t/action-advised-nerveshub-sunset/42925).
 This doesn't mean NervesHub development has ended. Many of us run NervesHub
-instances internally at our companies and really like it. We will be moving our
-patches to the source code to this `next` branch. The `main` branch continues to
-deploy to `nerves-hub.org`.
+instances internally at our companies and really like it. Development continues
+on the `main` branch, while the `main-next` branch will be deployed to `nerves-hub.org`
+through the sunset.
```
Comment (fhunleth) [inline README.md:16]:
```suggestion
on the `main` branch.
```
Simplified, since I'm not sure on the status of `main-next`.

Comment (fhunleth) [reply README.md:16]: Yeah, good point. Let's drop it.

File: README.md (hunk 1 above)
Comment (jjcarstens) [reply README.md:144]: Ya, probably just drop it for now. I have a little lib I use to fake devices but it needs more thought to be more easily usable.

Comment (dnrce) [reply README.md]: Dropped this section entirely.

**Review by fhunleth — APPROVED** [review]

**Comment by fhunleth** [comment]
Thanks!

**Pull request merged**